sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2374,14 +2374,14 @@ object functions {
    * @group datetime_funcs
    * @since 1.5.0
    */
-  def date_add(start: Column, days: Int): Column = withExpr { DateAdd(start.expr, Literal(days)) }
+  def date_add(start: Column, days: Int): Column = withExpr { AddDays(start.expr, Literal(days)) }
```

Comment:
then we have `date_add` to add days or interval to the given date, and `add_months` to add months to the given date, seems a little weird...